lib/node_modules/@stdlib/blas/base/gscal/lib/main.js

```diff
@@ -30,9 +30,9 @@ var ndarray = require( './ndarray.js' );
 * Multiplies `x` by a scalar `alpha`.
 *
 * @param {PositiveInteger} N - number of indexed elements
-* @param {number} alpha - scalar
+* @param {number} alpha - scalar constant
 * @param {NumericArray} x - input array
-* @param {PositiveInteger} stride - index increment
+* @param {PositiveInteger} strideX -  stride length
```

Comment:
```suggestion
* @param {PositiveInteger} stride -  stride length
```
Same suggestion throughout this file.